### DISCUSSION TITLE
vercel deploy local build result with Cannot find module 'next/dist/server/next-server.js'

### DISCUSSION THREAD

**Discussion by naorye** [vercel/vercel#9185] [Help]
I am trying to "vercel deploy" a build that was build locally but when I invoke dynamic route (serverless function) I get an error:
```
2023-01-07T20:31:52.337Z	undefined	ERROR	Cannot find module 'next/dist/server/next-server.js'
Require stack:
- /var/task/___next_launcher.cjs
```

background info:
I am using monorepo and one of the packages is the app I am trying to deploy.
Out of the box, `vercel build` didn't work and I had to change `rootDirectory` config in `.vercel/project.json` (set it to "/").
In addition I added the following to next.config.js:
```
 experimental: {
    outputFileTracingRoot: path.join(__dirname, "../../"),
  },
```
The commands I run are:
1. `vercel build --debug --prod`
2. `vercel deploy --prebuilt --prod --debug`

**Comment by amyegan** [comment]
Hi @naorye. Do you have a public repo or [minimal reproduction](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to get a better understanding of what is going wrong?
